### ISSUE TITLE
Issue with method "comments" of the Post class

### ISSUE THREAD

**Issue by seregahere** [timber/timber#1357]
### Expected behavior
Method "comments" of the Post class have $type parameter to filter comments by type.

### Actual behavior
$type variable is not used inside the "comments" method

### Suggestion
I suggest the following path:
```javascript
--- Post.php	2017-03-16 10:07:50.916293334 +0300
+++ updated/Post.php	2017-03-16 10:08:28.156627419 +0300
@@ -889,7 +889,7 @@
 		$commenter = wp_get_current_commenter();
 		$comment_author_email = $commenter['comment_author_email'];
 
-		$args = array('status' => $status, 'order' => $order);
+		$args = array('status' => $status, 'order' => $order, 'type' => $type);
 		if ( $count > 0 ) {
 			$args['number'] = $count;
 		}
```

**Comment by jarednova** [comment]
Thanks @seregahere — just created #1360 so I can verify and merge this in

**Comment by jarednova** [comment]
Closed by #1360
